Fix runtime.ini path section header name (#1440)
cyberbotics/webots: docs/guide/controller-programming.md

They will be automatically replaced by the actual value already existing in the environment.
The Webots "runtime.ini" supports 7 sections:

- `[environment variables with paths]`

    This section should only contain environment variables with either relative or
    absolute paths. Paths must be separated using the colon symbol ':' and the separator for

>     directories to use is the slash symbol '/'. Variables declared in this section will be

Reviewer: That seems simpler and clearer to me:
```suggestion
    absolute paths. Paths must be separated using the colon symbol ':' and directory components
    must be separated using the slash symbol '/'. Variables declared in this section will be
```

    added on every platform. On Windows, colons will be replaced by semicolon and

>     slash will be replaced by backslash according to the Windows syntax.<br>

Reviewer: We should never use `<br>` to add empty lines, instead, you can enter empty lines, like this:
```suggestion
    slash will be replaced by backslash according to the Windows syntax.

```

    **NOTE**: The legacy form of this section was ```[environment variables with relative path]```.

>     You are encouraged to follow the new notation, though the backward compability is kept.

Reviewer: I would skip this note (we usually don't keep track of previous versions)
```suggestion
```
